Rebus.Recipes/Configuration/CaptureIdentityConfigurationExtensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Rebus.Config;
+using Rebus.Pipeline;
+using Rebus.Pipeline.Receive;
+using Rebus.Pipeline.Send;
+
+namespace Rebus.Recipes.Identity
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public static class CaptureIdentityConfigurationExtensions 
+    {
+        /// <summary>
+        /// Propagates the ClaimsPrincipal through the message bus so that its there during message evaluation
+        /// </summary>
+        /// <param name="configurer"></param>
+        /// <returns></returns>
+        public static void AutomaticallyPropagateCurrentClaimsPrincipal(this OptionsConfigurer configurer)
+        {
+            configurer.AutomaticallyPropagateCurrentClaimsPrincipal(new DefaultClaimsPrinicpalSerializer());
+        }
+        /// <summary>
+        /// Propagates the ClaimsPrincipal through the message bus so that its there during message evaluation
+        /// </summary>
+        /// <param name="configurer"></param>
+        /// <param name="serializer"></param>
+        /// <returns></returns>
+        public static void AutomaticallyPropagateCurrentClaimsPrincipal(this OptionsConfigurer configurer,
+            IClaimsPrinicpalSerializer serializer)

```

Comment:
wondering why you are providing the serializer instance from the outside? in most cases (if not all) I would imagine that the serialization format of the claims would be pretty unimportant

it's not that this is a problem, but as a general principle I would like to always offer as few overloads as possible